static/app/components/organizations/environmentPageFilter/index.tsx

```diff
@@ -203,6 +203,7 @@ export function EnvironmentPageFilter({
       menuWidth={menuWidth ?? defaultMenuWidth}
       menuBody={desynced && <DesyncedFilterMessage />}
       menuFooterMessage={footerMessage}
+      menuWiderThanTrigger
```

Comment:
this prop means "menu should be wider than trigger" not "menu is already wider than trigger", i.e. when it's true we'll calculate the trigger's width when it changes. It's doing the opposite of what we seem to be trying to do (?)